calico_containers/pycalico/netns.py

```diff
@@ -283,3 +294,17 @@ class NamespaceError(Exception):
     Error creating or manipulating a network namespace.
     """
     pass
+
+
+class Namespace(object):
+    def __init__(self, path):
+        self.path_ = path
+
+    @property
+    def path(self):
+        return self.path_
```

Comment:
I don't think this gives us any benefits.  The init should just be `self.path = path`.